lib/util/qualifyUrlWithHost.js

```diff
@@ -4,7 +4,8 @@ var calculateSyncRequestUrl = require("../util/calculateSyncRequestUrl");
 
 var qualifyUrlWithHost = function(apiHost) {
```

Comment:
this file should probably be renamed to something like prepareUrlToBeSentToApiHost...but shorter